spec.bs

```diff
@@ -4292,13 +4336,36 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
         is not null;
       * |ig|'s [=interest group/estimated size=] is greater than 50 KB.
     1. Set |ig|'s [=interest group/next update after=] to the [=current wall time=] plus 24 hours.
+    1. Set |ig|'s [=interest group/last updated=] to the [=current wall time=].
     1. [=list/Replace=] the [=interest group=] that has |ig|'s [=interest group/owner=] and
       [=interest group/name=] in the [=user agent=]'s [=interest group set=] with |ig|.
     1. <i id=abort-update>Abort update</i>: We jump here if some part of the
       [=interest group=] update failed. [=iteration/Continue=] to the next [=interest group=] update.
 
 </div>
 
+<div algorithm>
+
+To <dfn>process updateIfOlderThanMs</dfn> given an [=origin=] |buyer|, and an [=ordered map=]
+|perInterestGroupData| whose [=map/keys=] are [=interest group/name=] [=strings=] and whose
+[=map/values=] are [=bidding signals per interest group data=]:
+
+1. [=map/For each=] |igName| → |perIgData| of |perInterestGroupData|, the following steps should be
+  done:
+  1. If |perIgData|'s [=bidding signals per interest group data/updateIfOlderThanMs=] is null,
+    [=iteration/continue=].
+  1. Let |ig| be the [=interest group=] of the [=user agent=]'s [=interest group set=] whose
+    [=interest group/owner=] is |buyer| and whose [=interest group/name=] is |igName|, or null if
+    [=interest group set=] does not have such an interest group.
+  1. If |ig| is not null and the [=current wall time=] minus |ig|'s [=interest group/last updated=]

```

Comment:
https://github.com/WICG/turtledove/pull/1124#discussion_r1593294289 (i.e., `&minus;` probably, or copy the symbol)

Reply:
Done.